examples/sample-input-byo.tfvars

```diff
@@ -105,7 +105,7 @@ node_pools = {
   }
 }
 
-# Jump Box
+# Jump Server
```

Comment:
We actually call it the 'Jump Box' here and in most docs. This will need to stay as 'Jump Box'